tests/README.md

```diff
@@ -0,0 +1,55 @@
+# Ansible FreeIPA Test Playbooks
+
+This directory contains a series of playbooks used to test the idempotency of roles provided by ansible-freeipa.
```

Comment:
I think changing "the idempotency of roles" to "modules and roles" make this more correct, as we test the modules functionality and not only idempotency.